packages/contractkit/src/wallets/ledger-types.d.ts

```diff
@@ -0,0 +1,83 @@
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
```

Comment:
This file a `d.ts.` looks like it's in a bad location.. shouldn't this be part of `/types` 
how would others refer to this?

Reply:
I found a really nice workaround. I removed it ;)